ui/scripts/docs.js

```diff
@@ -629,6 +629,10 @@ cloudStack.docs = {
         desc: 'Specify whether this offering is for a virtual private cloud',
         externalLink: ''
     },
+    helpNetworkOfferingTungsten: {
+        desc: 'Specify whether this offering is for tungsten SDN',
```

Comment:
Again, for end user facing docs, I would recommend using "tungsten fabric" instead of just "tungsten"